# HG changeset patch
# User jwe@segfault.lan
# Date 1201922767 18000
#      Fri Feb 01 22:26:07 2008 -0500
# Node ID abd750456740adc3e268ed2e9d571dfd0a3c07fd
# Parent  60f9ced8ab534e65f27d95a712d97ba6add6cd8d
Added tag ss-2-9-9 for changeset 60f9ced8ab53

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -17,8 +17,9 @@ 5c2f5830179082f18658d5a9e68087938444581e
 57b5030f5737ef9a1ac588b32e6c828358fac801 before-64-bit-merge
 9302581b820d71e6e3ae722b68e5923aebd46157 ss-2-9-2
 06585668a971b21818cacedbab1e11a98c11c5a0 ss-2-9-3
 f16c05db625059770b94eec0af027bbe74d2ce62 ss-2-9-4
 cda6a105ae9a30f9cd8032b1f49c2cfecfc04f6c before-ov-branch
 acb4a1e0b311c47bc957a48bf6af7f77e9c80fab ss-2-9-6
 b2e1be30c8e94902e6557c1edba2c4f37ff7bdf8 ss-2-9-7
 d542d919783961dce51ee8fc06f65b8f2cfe4156 ss-2-9-8
+60f9ced8ab534e65f27d95a712d97ba6add6cd8d ss-2-9-9
